### ISSUE TITLE
Help with SES account for Federalist

### ISSUE THREAD

**Issue by afeld** [18F/tts-tech-portfolio#72]
From @amirbey:

> a while back I was trying to setup a SES account ... i was wondering if you could help me out ... Erik generated an account for me before I left ... but i didn't get to register any domains/identities before he left ... and i don't seem to have the creds to do so ...
>
> ```
> "user": "ses-smtp-user.federalistapp-staging"
> ```
>
> the credentials i have are an accesskey, sercret and user (above)

@JJediny has been messing around in AWS more than I have, so volunteering him for this 😃 Let's get a short-term fix done, but then have a larger conversation about what AWS account(s) Federalist stuff should live in in the medium term.

**Comment by JJediny** [comment]
@afeld I'll pair up with @amirbey on the quick fix

**Comment by JJediny** [comment]
Update 
worked with @amirbey on a temp fix to setup a test SES account in sandbox to unblock them on dev/test work. 

TO DO:
My 2FA OTP is out of sync and need @afeld to assist me in resetting authenticator for the prod DNS account as SES service needs DNS verification; which requires the DNS record to be co-located in the same account as Route53/SES for `federalist.18f.gov` to unblock @amirbey implementation in prod setting.

**Comment by afeld** [comment]
Blocked by #76.

**Comment by JJediny** [comment]
Closing as development blocker is resolved will create a new issue for prod implementation
